tests/integration/targets/dcnm_vrf/tests/dcnm/query.yaml

```diff
@@ -82,13 +82,17 @@
 - assert:
     that:
     - 'result.changed == false'
-    - 'result.response[0].attach[0].deploy == true'
```

Comment:
@kharicha Quick question here.  Now that you have added the old fields back we can add these old tests back right?  Just want to make sure we have coverage for both old and new structure until we deprecate the old.

Reply:
yes we have added all the checks new ones and also the old ones,

- assert:
    that:
    - 'result.changed == true'
    - 'result.response[0].RETURN_CODE == 200'
    - 'result.response[1].RETURN_CODE == 200'
    - 'result.response[2].RETURN_CODE == 200'
    - '(result.response[1].DATA|dict2items)[0].value == "SUCCESS"'
    - '(result.response[1].DATA|dict2items)[1].value == "SUCCESS"'
    - 'result.diff[0].attach[0].deploy == true'
    - 'result.diff[0].attach[1].deploy == true'
    - '"{{ ansible_switch1 }}" in result.diff[0].attach[0].ip_address'
    - '"{{ ansible_switch2 }}" in result.diff[0].attach[1].ip_address'
    - 'result.diff[0].vrf_name == "ansible-vrf-int2"'

we can see that deploy from the old ones without parent/attach output is also checked.